app/webroot/js/track.js

```diff
@@ -0,0 +1,6 @@
+$("a.github_auth").click(function () {
+  ga('send', 'event', 'button', 'click', {'page': '/users/auth/github'})
```

Comment:
__[staging][eslint]__
*  Error - "ga" is not defined. (no-undef)

*  Error - Strings must use doublequote. (quotes)

*  Error - Strings must use doublequote. (quotes)

*  Error - Strings must use doublequote. (quotes)

*  Error - Strings must use doublequote. (quotes)

*  Error - Strings must use doublequote. (quotes)

*  Error - Strings must use doublequote. (quotes)

*  Error - Missing semicolon. (semi)